packages/checkout/sdk/src/sdk.ts

```diff
@@ -546,6 +546,7 @@ export class Checkout {
       web3Provider,
       itemRequirements,
       params.transactionOrGasAmount,
+      params.skipGasCalculation,
```

Comment:
Given this will be a new SDK property, can we discuss the most suitable name?
`skipGasCalculation` –> look fine at first, but the actual outcome is not around the calculation, but on not requirig gas checks

I think since gas is a requirement we may call it `skipGasRequirement` or `withoutGasFees` or something of the sort.

However, @dreamoftrees I just noticed, if we don't require to check for gas, then `transactionOrGasAmount` could be optional?

If thats the case, removing `transactionOrGasAmount` should make the gas fee optional

Reply:
Hmm, true. There is no point in passing `transactionOrGasAmount` if the gas check will be skipped anyway. Or passing GasAmount as 0, for example. @dreamoftrees to confirm